src/main/resources/anniv5Resources/localization/eng/coresetpack/Cardstrings.json

```diff
@@ -1,15 +1,15 @@
 {
   "${ModID}:BackpackSmack": {
     "NAME": "Backpack Smack",
-    "DESCRIPTION": "Deal !D! damage. NL If your draw pile has 10 or more cards, draw !M! cards. NL Otherwise, gain [E] ."
+    "DESCRIPTION": "Deal !D! damage X times. NL ${ModID}:Synergy: Gain [E] ."
   },
   "${ModID}:BoosterTutor": {
     "NAME": "Booster Tutor",
     "DESCRIPTION": "Put !M! random cards from different Packs from your draw pile into your hand."
   },
   "${ModID}:DestinyDraw": {
     "NAME": "Destiny Draw",
-    "DESCRIPTION": "Choose 1 of 3 random cards from different Packs to add to your hand. NL It costs 0 this turn. NL Exhaust."
```

Comment:
Why remove the word random here? Discovery and all other cards like it use "random" and I think it's an important word to keep in the text. It helps establish this as a parallel effect to Discovery that will be random just like it.

I'm fine with removing the NL (I see that Discovery doesn't have one), but I think "random" should stay.

Reply:
'random' is the difference between 5 lines and 6 lines in this case, that was the reasoning.  Perhaps we can find another shortening elsewhere?  Or leave it as is, but it really doesn't look good at 6 lines.

Reply:
It's my point of view that sticking with consistent wording is more important than avoiding 6 lines. It would be very strange to have this be the one card like this that doesn't say "random" (thus implying that its doing something different than everything else, even though it isn't). Especially given that the card in the current released version is already 6 lines.

If there's a way to fix it being six lines with breaking wording consistently, I'm all for it. Otherwise, I think keeping the status quo is better than fixing one problem only to introduce another one.

Reply:
On the topic of fixing it being six lines while keeping "random" -- what happens if you remove the NL before Exhaust? I believe there are examples of that approach in other mods (and maybe even somewhere in the base game?). And I like a minor formatting inconsistency like that a lot more than wording inconsistency.

Reply:
Was already trying that out, and it just baaaarely works.  So shall it be.